pkg/controller/controller.go

```diff
@@ -102,8 +105,36 @@ func NewController(clientset kubernetes.Interface, ssclientset ssclientset.Inter
 		},
 	})
 
+	sInformer := sinformer.Core().V1().Secrets().Informer()
+	sInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		DeleteFunc: func(obj interface{}) {
+			key, err := cache.MetaNamespaceKeyFunc(obj)
+			if err != nil {
+				return
+			}
+
+			ns, name, err := cache.SplitMetaNamespaceKey(key)
+			if err != nil {
+				return
+			}
+
+			ssecret, err := ssclientset.BitnamiV1alpha1().SealedSecrets(ns).Get(context.Background(), name, metav1.GetOptions{})
+			if err != nil {
+				return
+			}
+
+			key, err = cache.MetaNamespaceKeyFunc(ssecret)
+			if err != nil {
+				return
+			}
```

Comment:
I understand these are more checks that usual sources of errors, but still the fact that the function returns a `error` type and not a boolean means we need to check what error we are getting.

If the error is the expected, like `not found` then we can silently return. If the error is something else we should log it as it is unexpected.